App.js

```diff
@@ -0,0 +1,67 @@
+import { StatusBar } from 'expo-status-bar';
+import { StyleSheet, Text, View } from 'react-native';
+import {LinearGradient} from 'expo-linear-gradient'
+import { TextInput, TouchableWithoutFeedback } from 'react-native-web';
+import Button from './Component/Button';
+import React, { useState } from 'react';
+import SignIn from './Component/SignIn';
+import SignIn2 from './Component/SignIn2';
+import MainPage from './Component/MainPage';
+import { NavigationContainer, TabActions } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+
+const Stack = createNativeStackNavigator(); 
+
+const MyStack = () => {
+  return (
+      
+        <NavigationContainer >
+         <Stack.Navigator screenOptions={{headerTransparent: true}}
+         initialRouteName="ButtonLogin">
+          <Stack.Screen
+                name="Home"component={Button}/>
+          <Stack.Screen 
+                name="SignIn" component={SignIn} />
+          <Stack.Screen
+                name="MainPage" component={MainPage}/>
+         </Stack.Navigator>
+        </NavigationContainer>
+    
+  );
+};
+
+
+  function App() {
```

Comment:
Are you using this app component anywhere? Seems like you're using MyStack but not the App?

Reply:
I thought i was using it because that what gives the top of the screen the home and next section- was tryng to learn how to use stack navigator so not sure if i did it the right way i followed a youtube video and customised it